src/pages/TaskList/TaskList.js

```diff
@@ -0,0 +1,79 @@
+import './taskList.scss';
+import {Link} from 'react-router-dom';
+import {Tabs, Tab} from '../../components/Tabs/index';
+import { getTasks, deleteTask, updateTask } from '../../services/tasks';
+
+export class TaskList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      tasksInWeek: []
+    };
+  }
+  days = ['Sun', 'Mon', 'Thur', 'Wed', 'Th', 'Fr', 'St'];
+  date = new Date().getDay();
+
+  createTask = (day) => {
+    this.props.history.push(`tasks/newtask?day=${day}`);
+  };
+
+
+  handleDeleteTask = (id) => {
+    deleteTask(id)
+      .then(() => this.updateTaskList());
+  };
+
+  updateTaskList = () => {
+    getTasks()
+      .then(tasksInWeek => this.setState({tasksInWeek}));
+  };
+
+  handleDoneTask = (task) => {
+    task.done = true;
+    updateTask(task)
+      .then(() => this.updateTaskList());
+  };
+
+  handleInProgressTask = (task) => {
+    task.done = false;
+    updateTask(task)
+      .then(() => this.updateTaskList());
+  };
+
```

Comment:
`handleInProgressTask` and `handleDoneTask` make the same things. So, use one method instead of two